Portfolio: Desktop Version

JuanDiegoGuerra opened the pull request:
Updated index.html & mobile_first.css
 -
> Worked in a 768px view
 - Added remaining sections for my recent works.
 - Added nav menu in top of the page.
 - Fixed stlyes in buttons.
 - Implemented Media Queries for Desktop Version
 - Reorder some elements for Desktop Version
 - Added new background imgs for Desktop Version

> Sumary Preview :
![desktop-version-preview](https://user-images.githubusercontent.com/45726631/228674469-84f1683a-b0c1-4597-b345-727997f09693.gif)

JuanDiegoGuerra committed:
Update .gitignore

JuanDiegoGuerra committed:
Merge branch 'main' of https://github.com/JuanDiegoGuerra/Portfolio-Setup-and-Mobile-first

JuanDiegoGuerra committed:
Update index.html

Added some sections and also fixed others that were not working properly.

JuanDiegoGuerra committed:
Add Images for desktop-version background

8 images added. 1 deleted/modified for a new one.

JuanDiegoGuerra committed:
Update mobile_first.css

Implemented Media Queries styles for the desktop version.

JuanDiegoGuerra committed:
Merge branch 'main' into Portfolio-Desktop-Version

iambenkis commented on index.html:
```diff
@@ -29,18 +29,19 @@
         </nav>
         </header>
         <section id="headline-section">
-                <h1 class="headline-title"> Hi there! <br>
```
- [ ] Good job adding the desktop version to your portfolio 👏 , but I noticed that the hamburger button that you use in the mobile version is still present in your desktop version, kindly remove it 👍 

![Capture 5](https://user-images.githubusercontent.com/80458199/228805882-43da6d03-713b-495f-9831-51f15ca97805.PNG)

iambenkis commented on index.html:
```diff
@@ -90,20 +92,102 @@ <h3>
                     <a href="#">See Project</a>
                 </div>
             </div>
-        </div>       
+        </div>
+        <div class="project-2">
+            <div class="first_project_img_div">
+            <img src="img/default_project_img.JPG" alt="first project image" class="first_project_image">
+            </div>
+            <div class="first-project">
+              <h3>
+                Setup-Portfolio-Mobile-1st <br>
```
- [ ] The projects section should be center , Kindly center this section to stick to the Figma design and come out with a beautiful design 👍 

![Capture 3](https://user-images.githubusercontent.com/80458199/228806380-559c6e85-7f06-47a1-8544-c1e018ef4236.PNG)

iambenkis commented on index.html:
```diff
@@ -90,20 +92,102 @@ <h3>
         <div class="about-me-bg">
-        <h3 class="about-me-tittle">About me</h3>
-        <div class="parraf">
-        <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
```
- [ ] As shown in the Figma design , the about me section , mostly the part in your portfolio where you listed the language , the skills, and the framework, it could be better if you center them. It will make your design stick to the figma design 💯 

| Your design |  Figma design |
| :---: | :---: |
| ![Capture 2](https://user-images.githubusercontent.com/80458199/228807187-4149f85e-aa63-4efc-9749-4c9fc9874845.PNG) |  ![Capture 1](https://user-images.githubusercontent.com/80458199/228807377-27f5244a-209f-4168-8638-8393d2a4287f.PNG) |

iambenkis commented on index.html:
```diff
@@ -153,35 +237,49 @@ <h3 id="h3-lang">
+           <textarea class="txtS" name="message" rows="4" maxlength="500" required placeholder="Hey I’d like to connect to..."></textarea>
            <button type="submit" name="submit" id="contact-button">Get in touch</button>
         </form>
+        </div>
+        <div class="inputs-form2">
+            <form class="contact" action="https://formspree.io/f/mqkoyzva" method="post">
+                <div class="first-last-name">
+                <input class="first-name" type="text" name="name" placeholder="First Name" maxlength="15" required>
+                <input class="last-name" type="text" name="name" placeholder="Last Name " maxlength="15" required>
+                </div>
+                <input type="email" name="email" placeholder="Email address" required>
+                <textarea class="txtS" name="message" rows="4" maxlength="500" required placeholder="Hey I’d like to connect to..."></textarea>
+                <button type="submit" name="submit" id="contact-button">Get in touch</button>
+             </form>
```
- [ ] Kindly update the width of this contact form so it sticks to the Figma design 👍 

![Capture 4](https://user-images.githubusercontent.com/80458199/228807857-ff1a8fcf-b6dc-435e-a0bf-369787eb07af.PNG)

iambenkis reviewed (changes requested):
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @JuanDiegoGuerra 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

![almost-there](https://user-images.githubusercontent.com/57562869/143511358-9cde7daf-4ad0-4db2-8f15-18b54288ff91.gif)

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 

- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @iambenkis  when you comment, so I can receive the notification or use Slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

JuanDiegoGuerra committed:
Update index.html

Added a container in order to use flex grip and fix positions of what I was required.

JuanDiegoGuerra committed:
Update mobile_first.css

- Implemented styles for the container added in last commit.
- Removed unnecessary styles.
- Fixed linters error.

JuanDiegoGuerra committed:
Merge branch 'Portfolio-Desktop-Version' of https://github.com/JuanDiegoGuerra/Portfolio-Setup-and-Mobile-first into Portfolio-Desktop-Version

JuanDiegoGuerra committed:
Update mobile_first.css

Fixed new linter errors and removed unnecessary comments.

JuanDiegoGuerra commented:
Updated index.html & mobile_first.css
 -
> Made the Required Changes: ♻
 - Removed hamburguer button when displaying Desktop View. ✅
 - Fixed "projects" positions from "my-work" section. ✅
 - Fixed positions from "languages", "frameworks" and "skills" sections. ✅
 - Fixed width of the contact form. ✅

> Sumary Preview : 🕵🏼‍♂️

![desktop_finalversion_preview](https://user-images.githubusercontent.com/45726631/228972352-5064a56e-2789-4459-a5fd-4ae8334d5549.gif)

tufoinnkuo10 commented on index.html:
```diff
@@ -153,35 +237,51 @@ <h3 id="h3-lang">
     <div class="project-1" id="about-me-section-4">
+        <div class="input-container">
         <div class="am-contact-me-main-grid">
          <h2 class="contact-title">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
```
- [optional] great job making the changes. kindly use a p tag here as it is not a heading
- [optional]  please kindly use font-weight: 700;///font-size: 30px;///line-height: 52px so the text follow the figma design
- [optional] kindly ensure that the text and  contact form aligns with the cards above it
yours

![Screenshot (920)_LI](https://user-images.githubusercontent.com/65298048/228976824-4bd68ad0-b7b0-4d21-b860-0b27ad88cd36.jpg)

figma
![Screenshot (919)](https://user-images.githubusercontent.com/65298048/228976916-6decca46-faca-4285-a332-cbda3a5286e5.png)

tufoinnkuo10 reviewed (approved):
## Status: Approved :heavy_check_mark:

Hi @JuanDiegoGuerra 

Your project is complete! Great job for making the changes requested by the previous reviewer 🥇.please take the optional changes seriously. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- Figma design followed
- responsive
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

![tufoin](https://media.giphy.com/media/3o84TV5bThVrRvyDRe/giphy.gif)

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
